src/index.html

```diff
@@ -1,15 +1,221 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="highest groups">
+        <legend>Personal information</legend>
+
+        <div class="highest questions">
+          <label for="surname">
+            Surname:
+            <input
+              id="surname"
+              type="text"
+              name="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="questions">
+          <label for="name">
+            Name:
+            <input
+              id="name"
+              type="text"
+              name="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="questions">
+          <label for="age">
+            How old are You?
+            <input
+              id="age"
+              type="number"
+              name="age" min="5"
```

Comment:
```suggestion
              name="age" 
              min="5"
```
кожен атрибут повинен бути на новому рядку